core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -304,6 +305,11 @@ class SparkContext(config: SparkConf) extends Logging {
     _dagScheduler = ds
   }
 
+  private[spark] def heartbeater: Heartbeater = _heartbeater
+  private[spark] def heartbeater_=(hb: Heartbeater): Unit = {
```

Comment:
I don't think you're using this getter and setter at all?

Reply:
These aren't used -- I'll remove.